vlm/src/main/scala/geotrellis/contrib/vlm/RasterSourceRDD.scala

```diff
@@ -115,6 +115,61 @@ object RasterSourceRDD {
     ContextRDD(result, layerMetadata)
   }
 
+  def readFromRDD(
```

Comment:
a bit a confusing name(?)

Reply:
Yeah, it's not the best name.

Reply:
I was thinking `read` since it's dealing with `ReadingSource`s. Regardless, though, the reason I didn't do that, though, is because of:

```
[error] /home/jake/Documents/projects/geotrellis-contrib/vlm/src/main/scala/geotrellis/contrib/vlm/RasterSourceRDD.scala:33:8: in object RasterSourceRDD, multiple overloaded alternatives of method read define default arguments.
[error] object RasterSourceRDD {
```

So I'm going to have to figure out a way to refactor those methods so that they can have the same name.